core/vm/contracts.go

```diff
@@ -122,11 +129,12 @@ var PrecompiledContractsBLS = map[common.Address]PrecompiledContract{
 }
 
 var (
-	PrecompiledAddressesCancun    []common.Address
-	PrecompiledAddressesBerlin    []common.Address
-	PrecompiledAddressesIstanbul  []common.Address
-	PrecompiledAddressesByzantium []common.Address
-	PrecompiledAddressesHomestead []common.Address
+	PrecompiledAddressesP256Verify []common.Address
```

Comment:
do we need this? looks like we include those addresses in `PrecompiledAddressesArbOS30` instead

Reply:
Removed